### PR TITLE
MAX-11493 Icon and Words are Too Closed in App Switcher

### PR DESCRIPTION
**Pull request by benjaminliugang**
* Fixed Dropdown item icon spacing issue.
* Removed useless class name.

image

@joshsylvester @sheela-svmx , please help review.

### CODE REVIEW

**Comment by shao** [comment]
@joshsylvester @sheela-svmx cc @sabrinalui-ge 

No action from UI team?

**Comment by sheela-svmx** [comment]
@benjaminliugang  this was merged on the main repo correct. I saw Josh reviewed and had comments and it was fixed in the other repo?  Are we using both the fork and the Main px-dropdown?

File: px-dropdown-content.html [hunk 1]
```diff
@@ -81,7 +81,7 @@
                 <px-icon icon="px-utl:check" class="select-icon u-pr--"></px-icon>
               </template>
               <template is="dom-if" if="{{item.icon}}">
-                <px-icon icon="{{item.icon}}" class="item-icon u-mr-" style$="color:{{item.color}};stroke:{{item.color}}"></px-icon>
+                <px-icon icon="{{item.icon}}" class="item-icon" style$="color:{{item.color}};stroke:{{item.color}}"></px-icon>
```
Comment (sheela-svmx) [inline px-dropdown-content.html]: Also @benjaminliugang  this change was reverted in the main px-dropdown PR , https://github.com/predixdesignsystem/px-dropdown/pull/100/files 

This just has to match the one that was merged there.

**Comment by benjaminliugang** [comment]
It seems we already switch to the main repo in [ui-components-predix](https://github.com/ServiceMax-Engineering/ui-components-predix/blob/master/bower.json#L11) last week, just  close this PR, let's focus on the main repo.

**Pull request closed without merging**